src/vs/workbench/services/telemetry/electron-browser/telemetryService.ts

```diff
@@ -33,7 +34,10 @@ export class TelemetryService extends Disposable implements ITelemetryService {
 	) {
 		super();
 
-		if (!environmentService.isExtensionDevelopment && !environmentService.args['disable-telemetry'] && !!productService.enableTelemetry) {
+		if (environmentService.args['perf-test']) {
+			let telemetryOutput = environmentService.args['telemetry-output'];
+			this.impl = this._register(new FileTelemetryService(telemetryOutput));
+		} else if (!environmentService.isExtensionDevelopment && !environmentService.args['disable-telemetry'] && !!productService.enableTelemetry) {
 			const channel = sharedProcessService.getChannel('telemetryAppender');
 			const config: ITelemetryServiceConfig = {
 				appender: combinedAppender(new TelemetryAppenderClient(channel), new LogAppender(logService)),
```

Comment:
Here it appears that are outputting telementry to the logservice, which outputs to a file on disk. Could you just use that file instead?